sc/social/like/subscribers.py

```diff
@@ -151,3 +152,36 @@ def check_sharing_best_practices(obj, event):
         validate_og_lead_image(image)
     except ValueError as e:
         api.portal.show_message(message=e.message, request=request, type='warning')
+
+
+def prefetch_facebook(obj, event):
+    """Prefetching in object if enable."""
+
+    record = ISocialLikeSettings.__identifier__ + '.facebook_prefetch_enable'
+    prefetch_enable = api.portal.get_registry_record(record, default=False)
+
+    if not prefetch_enable:
+        return
+
+    try:
+        review_state = api.content.get_state(obj)
+    except WorkflowException:
+        # images and files have no associated workflow by default
+        review_state = 'published'
+
+    if review_state not in ('published', ):
+        return  # no need to validate
+
+    url = obj.absolute_url()
```

Comment:
IMO, it will be better to use:

```python
url = 'https://graph.facebook.com/?id=' + obj.absolute_url() + '&scrape=true'
r = requests.post(url, timeout=5)
```

Reply:
Okay